src/core/mwaa-environment.ts

```diff
@@ -144,7 +144,7 @@ export class MWAAEnvironment extends Construct {
       }
     }
 
-    const mwaaExecutionRole = new iam.Role(scope, "MWAA Execution Role", {
+    const mwaaExecutionRole = new iam.Role(scope, `${props.name} MWAA Execution Role`, {
```

Comment:
This pattern will need to be followed for all resources, (i.e. `s3deploy.BukcketDeployment()`, etc.

Reply:
Hi, I have added the prefix for all resources in the stack. Regarding the vpc name, you mentioned it was okay i used the resourceName that has -mwaa at the end, but let me know if i should revert to using props.name directly from the function args to make it same accross all resources. 

I have also created a test in the mwaa-environment.test.ts that check if there are 2 MWAAEnvironments created. 

Please let me know if any additional modifications or tests are required.